src/deployment/flavors.md

```diff
@@ -3,41 +3,226 @@ title: Creating flavors for Flutter
+For example, you might have one flavor for your full-fledged production app, 
+another as a limited "free" app, another for testing experimental features, and so on. 
+
+Let's say you want to make both free and paid versions of your Flutter app. 
+You can use flavors to set up both app versions 
+without writing two separate apps. 
+The free version of the app has basic functionality and ads (for example). 
+In contrast, the paid version has basic app functionality, extra features, 
+different styles for paid users, and no ads. 
+
+You also might use flavors for feature development. 
+If you’ve built a new feature and want to try it out, 
+you could set up a flavor to test it out. 
+Your production code remains unaffected 
+until you're ready to deploy your new feature.
+
+Flavors let you define compile-time configurations 
+and set parameters that are read at runtime to customize
+your app's behavior.
+
+This document guides you through setting up Flutter flavors for iOS and Android. 
+
+## Environment set up
+Prerequisites:
+* Xcode installed
+* An existing project 
+
+To set up flavors in iOS, you’ll define build configurations in Xcode. 
+
+## Creating flavors in iOS
+
+<ol markdown="1">
+<li markdown="1">
+
+Open your project in Xcode.
+
+</li>
+<li markdown=1>
+
+Select **Runner** > **New Scheme** from the menu to add a new `Scheme`.  
+* A scheme describes how Xcode runs different actions. 
+  For the purposes of this guide, the example _flavor_ and _scheme_ are 
+  named `free`. 
+  The build configurations in the `free` scheme 
+  have the `-free` suffix. 
+
+</li>
+<li markdown="1">
+
+Duplicate the build configurations to differentiate between the 
+default configurations that are already available and the new configurations 
+for the `free` scheme. 
+* Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name (Debug, Release, and Profile). 
+Duplicate the listed production configurations, once for each environment. 
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+{{site.alert.note}}
+  Your configurations should be based on your **Debug.xconfig** or **Release.xcconfig**
+  file, not the **Pods-Runner.xcconfigs**. You can check this by expanding the configuration names. 
+{{site.alert.end}}
+
+</li>
+<li markdown="1">
+
+To match the development flavor, add `-free` 
+at the end of each new configuration name. 
+
+</li>
+<li markdown="1">
+
+Change the `free` scheme to match the development 
+build configurations already created.
+* In the **Runner** project, click **Manage Schemes…** and a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
```

Comment:
These images need to be updated to show `-free` instead of `-production` and `-development`.

![Screenshot 2022-12-21 at 10 47 31 AM](https://user-images.githubusercontent.com/682784/208980856-f562180e-3c52-4b1c-90a7-2984c52299b2.png)

Reply:
Yes, I am in the process of updating them.